### DISCUSSION TITLE
How to have round corners with borders in dwm?

### DISCUSSION THREAD

**Discussion by Randomguy-8** [bakkeby/dwm-flexipatch#306] [How To]
I love round corners but i want borders too...

I tried **roundcorners** patch but it disables borders and if i use compositor (picom --round-corners) the borders get cut.

so how exactly i can get both borders and round corners in dwm?

**Comment by bakkeby** [comment]
Removing the border size is how the rounded corners patch was intended to be used, but you can set a border if you want to. The borders will be cropped similar to picom I believe (as in the inner window will still have hard edges).

I _think_ the [jonaburg](https://github.com/jonaburg/picom) version of picom did something extra with regards to this in that it would draw an antialiased border around the rounded edge, judging from the screenshot in [this reddit post](https://www.reddit.com/r/UsabilityPorn/comments/k8s8ph/qtile_i_showed_off_my_bspwm_rice_earlier_so_heres/). I assumed that this had been carried over to the main yshui picom version but perhaps that is not the case.

**Reply by amazingwilf** [reply]
I use the jonaburg picom to get rounded corners, and it's pretty good with borders of 2px or more.
